packages/clay-form/src/__tests__/InputWithMultiSelect.tsx

```diff
@@ -128,7 +128,7 @@ describe('Interactions', () => {
 		expect(document.activeElement!.tagName).toEqual('BODY');
 
 		fireEvent.keyDown(
-			container.querySelector('input') as HTMLInputElement,
+			container.querySelector('input.form-control-inset') as HTMLInputElement,
```

Comment:
This smells a bit implementation-detailish. Might be better to target by type eg.

```javascript
container.querySelector('input[type=text]')
```